transform/binary-plugin/binary_plugin.go

```diff
@@ -41,28 +67,66 @@ func (b *BinaryPlugin) Run(u *unstructured.Unstructured) (transform.PluginRespon
 	err = json.Unmarshal(out, &p)
 	if err != nil {
 		b.log.Errorf("unable to decode json sent by the plugin")
-		return p, fmt.Errorf("unable to decode json sent by the plugin: %s, err: %v", string(out), err)
+		return p, fmt.Errorf("unable to decode json for metadata sent by the plugin: %s, err: %v", string(out), err)
```

Comment:
I am confused why "for metadata" here? this is the response from the `Run` right?